libV2/utils/timestamp.js

```diff
@@ -32,7 +32,18 @@ class InterpolatedClock {
     }
 }
 
+/**
+ * Returns the current time as
+ * the number of microseconds since the epoch
+ *
+ * @returns {Number} - current timestamp
+ */
+function now() {
+    return new Date().getTime() * 1000;
```

Comment:
minor: I usually use `Date.now()` to get the epoch in milliseconds which is slightly more compact.